src/Microsoft.ML.Data/Scorers/MultiClassClassifierScorer.cs

```diff
@@ -327,7 +327,11 @@ public ISchemaBindableMapper Bindable
                     return builder.GetSchema();
                 }
 
-                public Func<int, bool> GetDependencies(Func<int, bool> predicate) => _mapper.GetDependencies(predicate);
+                /// <summary>
+                /// Given a set of columns, return the input columns that are needed to generate those output columns.
+                /// </summary>
+                IEnumerable<Schema.Column> IRowToRowMapper.GetDependencies(IEnumerable<Schema.Column> dependingColumns)
+                => _mapper.GetDependencies(dependingColumns);
```

Comment:
> => [](start = 15, length = 3)

somehow this triggers me. can you add tab?

Reply:
I will add 4 spaces ;)

---
In reply to: [256133529](https://github.com/dotnet/machinelearning/pull/2504#discussion_r256133529) [](ancestors = 256133529)